src/nrniv/nrnpy.cpp

```diff
@@ -272,15 +278,18 @@ static nrnpython_reg_real_t load_nrnpython() {
         auto const iter =
             std::find(supported_versions.begin(), supported_versions.end(), pyversion);
         if (iter == supported_versions.end()) {
-            std::cerr << "Python " << pyversion
-                      << " is not supported by this NEURON installation (supported:";
+            Fprintf(
+                stderr,
+                fmt::format("Python {} is not supported by this NEURON installation (supported:",
+                            pyversion)
+                    .c_str());
             for (auto const& good_ver: supported_versions) {
-                std::cerr << ' ' << good_ver;
+                Fprintf(stderr, fmt::format(" {}", good_ver).c_str());
             }
-            std::cerr << "). If you are seeing this message, your environment probably contains "
-                         "NRN_PYLIB, NRN_PYTHONEXE and NRN_PYTHONVERSION settings that are "
-                         "incompatible with this NEURON. Try unsetting them."
-                      << std::endl;
+            Fprintf(stderr,
+                    "). If you are seeing this message, your environment probably contains "
+                    "NRN_PYLIB, NRN_PYTHONEXE and NRN_PYTHONVERSION settings that are "
+                    "incompatible with this NEURON. Try unsetting them.");
```

Comment:
Couldn't this just be one call to `Fprintf`?

Reply:
Not yet, soon